.goreleaser.yml

```diff
@@ -50,14 +50,66 @@ archives:
     <<: *archive_defaults
     wrap_in_directory: false
     format: zip
+aurs:
+  -
+    homepage: "https://github.com/fastly/cli"
+    description: "A CLI for interacting with the Fastly platform"
+    maintainers:
+      - 'oss@fastly.com'
+    license: "Apache license 2.0"
+    skip_upload: auto
+
+    # The SSH private key that should be used to commit to the Git repository.
+    # This can either be a path or the key contents.
+    #
+    # WARNING: do not expose your private key in the config file!
+    private_key: '{{ .Env.AUR_KEY }}'
+
+    # The AUR Git URL for this package.
+    # Defaults to empty.
+    git_url: 'ssh://aur@aur.archlinux.org/cli-bin.git'
+
+    # List of packages that are not needed for the software to function,
+    # but provide additional features.
+    #
+    # Must be in the format `package: short description of the extra functionality`.
+    #
+    # Defaults to empty.
+    optdepends:
+      - 'viceroy: for running service locally'
+        #
+    # Custom package instructions.
+    #
+    # Defaults to `install -Dm755 "./PROJECT_NAME" "${pkgdir}/usr/bin/PROJECT_NAME",
+    # which is not always correct.
+    #
+    # We recommend you override this, installing the binary, license and
+    # everything else your package needs.
+    package: |-
```

Comment:
I don't know if this is needed, but they suggest it _should_ be updated.

Reply:
I don't think we should set this unless we need to override it. 

Defaults to `install -Dm755 "./PROJECT_NAME" "${pkgdir}/usr/bin/PROJECT_NAME"` seems pretty good to me to start.